projects/demo/src/modules/components/input/examples/6/index.html

```diff
@@ -44,6 +44,7 @@
                 <td tuiTd>
                     <tui-textfield tuiChevron>
                         <select
+                            aria-label="color"
```

Comment:
We should probably internalize it based on `placeholder` and aria-placeholder?

Reply:
I'm not sure, the placeholder and the label may be different